frontend/javascripts/oxalis/view/right-border-tabs/starting_job_modals.tsx

```diff
@@ -297,7 +303,19 @@ function StartingJobModal(props: StartingJobModalProps) {
     initialOutputSegmentationLayerName || "segmentation"
   }_corrected`;
   // TODO: Other jobs also have an output segmentation layer. The names for these jobs should also be configurable.
-  const hasOutputSegmentationLayer = jobName === "materializing this volume annotation";
+  const hasOutputSegmentationLayer = jobName === JobNames.MATERIALIZE_VOLUME_ANNOTATION;
+  const notAllowedOutputLayerNames = allLayer
+    .filter((layer) => {
+      // Filtering out the layer that is currently selected as this name is allowed
+      // because the output layer overwrites the selected layer.
```

Comment:
```suggestion
      // Filtering out the layer that is currently selected as this name is not allowed
      // because the output layer would overwrite the selected layer.
```

if I understood it correctly

Reply:
No sorry, that is exactly not the case here.
All layer names are forbidden !except! the name of the selected volume layer, because this volume layer will don't exist in the output dataset, but it will be replaced with the modified version of this layer. The modified version then will have the name given by the `OutputSegmentationLayerNameFormItem`.

Reply:
As you misunderstood this comment, do you have a different way to express this, that might be easier to understand / not misunderstand?